src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -31,30 +32,46 @@ public void postTenant(TenantAttributes tenantAttributes, Map<String, String> he
         return;
       }
 
-      if (isLoadSample(tenantAttributes)) {
-        TenantLoading tl = new TenantLoading();
-        tl.withKey(PARAMETER_LOAD_SAMPLE)
-          .withLead("data")
-          .add("organizations", "organization-storage/organizations")
-          .add("categories", "organization-storage/categories")
-          .add("contacts","organization-storage/contacts")
-          .perform(tenantAttributes, headers, vertx, res1 -> {
-            if (res1.failed()) {
-              hndlr.handle(io.vertx.core.Future.succeededFuture(PostTenantResponse
-                .respond500WithTextPlain(res1.cause().getLocalizedMessage())));
-              return;
-            }
+      TenantLoading tl = new TenantLoading();
+      boolean loadData = buildDataLoadingParameters(tenantAttributes, tl);
+
+      if (loadData) {
+        tl.perform(tenantAttributes, headers, vertx, res1 -> {
+          if (res1.failed()) {
             hndlr.handle(io.vertx.core.Future.succeededFuture(PostTenantResponse
-              .respond201WithApplicationJson("")));
-          });
+              .respond500WithTextPlain(res1.cause().getLocalizedMessage())));
+            return;
+          }
+          hndlr.handle(io.vertx.core.Future.succeededFuture(PostTenantResponse
+            .respond201WithApplicationJson("")));
```

Comment:
is there a method for returning 201 w/o response body?

Reply:
The PostTenantResponse has both 200 and 201 with application Json, the only status code without content is 204